assets/app/views/project.html

```diff
@@ -149,9 +149,11 @@ <h2 ng-if="!displayRouteByService[serviceId]">
                             and {{numRemaining}} {{numRemaining == 1 ? "other" : "others"}}
                           </span>
                         </span>
+                        <div class="spacer">&nbsp;</div>
```

Comment:
Totally agree.  Ideally I would have made "service-links" below a UL and then just changed the 'create route' link (Think it was a span originally) accordingly.  However, I don't think there's a way currently to have static markup within an element that's managed by the extension-point directive, i.e. extensions will overwrite anything I put in there.